### PR TITLE
Add Back Exp2 Fog

### PR DESCRIPTION
**Pull request by DoobesURU**
Reintroduces the Exp2 Fog option for completion and ZLZ importation reasons and also updates the "none" fog function description.

### CODE REVIEW

**Review by dpogue — APPROVED** [review]
:+1: from me

File: korman/properties/prop_world.py [hunk 1]
```diff
@@ -32,7 +32,8 @@ class PlasmaFni(bpy.types.PropertyGroup):
                               items=[
                                      ("linear", "Linear", "Linear Fog"),
                                      ("exp", "Exponential", "Exponential Fog"),
-                                     ("none", "None", "Use fog from the previous age")
+                                     ("exp2", "Exponential 2", "Exponential Fog 2"),
```
Comment (Hoikas) [inline korman/properties/prop_world.py:35]: I wonder if this should mention that it's distance squared?

Comment (DoobesURU) [reply korman/properties/prop_world.py:35]: Is that what it does? I honestly have no clue. 😜

Comment (DoobesURU) [reply korman/properties/prop_world.py]: @Hoikas I've updated the descriptions on all types. Thoughts?

File: korman/properties/prop_world.py [hunk 2]
```diff
@@ -30,9 +30,9 @@ class PlasmaFni(bpy.types.PropertyGroup):
                                     subtype="COLOR")
     fog_method = EnumProperty(name="Fog Type",
                               items=[
-                                     ("linear", "Linear", "Linear Fog"),
-                                     ("exp", "Exponential", "Exponential Fog"),
-                                     ("exp2", "Exponential 2", "Exponential Fog 2"),
+                                     ("linear", "Linear", "Fog Based on Line of Sight"),
+                                     ("exp", "Exponential", "Fog Based on Avatar Position"),
+                                     ("exp2", "Exponential 2", "Exponential Fog With Distance Squared"),
```
Comment (dpogue) [inline korman/properties/prop_world.py:35]: Better descriptions might be:

* **Linear:** Fog based on linear distance
* **Exponential:** Fog based on exponential distance
* **Exponential Squared:** Fog based on exponential distance squared

All fog is based on the distance from the camera to a vertex, with the following equations:

* **Linear**: `(fogEnd - camToVertexDistance) / (fogEnd - fogStart)`
* **Exponential:** `e ^ -(fogDensity * camToVertexDistance)`
* **Exponential Squared:** `e ^ -((fogDensity * camToVertexDistance) ^ 2)`

Comment (DoobesURU) [reply korman/properties/prop_world.py:35]: @dpogue Good enough for me. I'll make the changes.

**Review by Hoikas — APPROVED** [review]

**Pull request merged**